awx/playbooks/project_update.yml

```diff
@@ -148,18 +126,29 @@
         register: doesRequirementsExist
 
       - name: fetch galaxy roles from requirements.yml
-        command: ansible-galaxy install -r requirements.yml -p {{project_path|quote}}/roles/
+        command: ansible-galaxy install -r requirements.yml -p {{roles_destination|quote}}
         args:
           chdir: "{{project_path|quote}}/roles"
         register: galaxy_result
-        when: doesRequirementsExist.stat.exists and (scm_version is undefined or (git_result is not skipped and git_result['before'] == git_result['after']))
+        when: doesRequirementsExist.stat.exists
         changed_when: "'was installed successfully' in galaxy_result.stdout"
 
-      - name: fetch galaxy roles from requirements.yml (forced update)
-        command: ansible-galaxy install -r requirements.yml -p {{project_path|quote}}/roles/ --force
+      when: roles_enabled|bool
+      delegate_to: localhost
+
+    - block:
+      # TODO: don't run this if ansible verison doens't support collections
```

Comment:
Can we just lose this TODO now?

https://github.com/ansible/awx/pull/4265/files#diff-9d4ea1dd908b35fb92eaede4bd10bb46R1893